dialect.go

```diff
@@ -181,7 +181,7 @@ func (m TiDBDialect) insertVersionSQL() string {
 }
 
 func (m TiDBDialect) dbVersionQuery(db *sql.DB) (*sql.Rows, error) {
-	rows, err := db.Query(fmt.Sprintf("SELECT version_id, is_applied from %s ORDER BY id DESC", TableName()))
+	rows, err := db.Query(fmt.Sprintf("SELECT version_id, is_applied from %s ORDER BY tstamp DESC", TableName()))
```

Comment:
This wouldn't work for migrations that were applied at the same time.

I found multiple rows in goose_db_version table where `ts` is the same in our production database.